dags/hivemind_telegram_etl.py

```
@@ -0,0 +1,104 @@
+import logging
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.decorators import task
+from dotenv import load_dotenv
```

Ruff

`dotenv.load_dotenv` imported but unused

Remove unused import: `dotenv.load_dotenv`

(F401)